common/mariadb/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.20.0 - 2025/04/03
+* add `renameCheckConstraints`  job, which allows to rename constraints, created by sqlalchemy with names like `CONSTRAINT_*`, to constraints with a unique name, as MySQL does
+
```

Comment:
I guess a small explanation where this is needed would be helpful.

Reply:
I've added an additional context to a changelog.